lib/css_parser/parser.rb

```diff
@@ -391,7 +391,16 @@ def parse_block_into_rule_sets!(block, options = {}) # :nodoc:
             current_media_query = String.new
           else
             token.strip!
-            current_media_query << token << ' '
+            # special-case the ( and ) tokens to remove inner-whitespace
+            # (eg we'd prefer '(width: 500px)' to '( width: 500px )' )
+            case token
+            when '('
+              current_media_query << token
+            when ')'
+              current_media_query.sub!(/ ?$/, token)
+            else
+              current_media_query << token << ' '
+            end
           end
```

Comment:
This isn't necessary to fix the bug, but does give nicer media query names. It could be skipped if we're happy with

```css
@media(min-width: 500px) {...}
```

getting re-written to 

```css
@media ( min-width: 500px ) {...}
```

I'm happy with either, but it's maybe not worth the extra complexity. Without it, I thought the test suite needing the extra spaces in, eg, `@cp.find_by_selector('body', :'( min-width: 500px )')` was kind of ugly... but I don't know how much real-world usage the media_types argument to find_selector is getting.